pkgs/development/python-modules/python3-msrplib/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, pythonOlder
+, buildPythonPackage
+, fetchFromGitHub
+, python3-application
+, python3-eventlib
+, python3-gnutls
+, twisted
+}:
+
+buildPythonPackage rec {
+  pname = "python3-msrplib";
+  version = "0.21.0";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "AGProjects";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-rjkg6iRvnokNikd2UAyngKuS2tkeKmGdu4Uvw5lvcys=";
+  };
+
+  propagatedBuildInputs = [
+    python3-application
+    python3-eventlib
+    python3-gnutls
+    twisted
+  ];
+
```

Comment:
Whats with the tests https://github.com/AGProjects/python3-msrplib/tree/master/test ?